README.md

```diff
@@ -583,6 +584,31 @@ export default function PostForm() {
 }
 ```
 
+NodeJS example:
+
+```js
+const client = new GraphQLClient({
+  url: 'https://domain.com/graphql',
+  cache: memCache(),
```

Comment:
We can remove this - it's not required for this example and may confuse people.

Reply:
Ok, done